ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -69,6 +69,15 @@ private[state] object Conversions {
       .build
   }
 
+  def resolvedContractKeyIdPair(
+      contractKey: DamlContractKey,
+      contractId: Option[ContractId]): DamlContractKeyIdPair =
+    DamlContractKeyIdPair.newBuilder
+      .setKey(contractKey)
+      // an empty string is used to mark "no contract id"
```

Comment:
I'd rather document this in the protobuf definition itself because that's where I'd be looking for such documentation.

Reply:
Done.